src/Utils/request/errorHandler.ts

```diff
@@ -87,12 +110,16 @@ function isPydanticError(errors: unknown): errors is PydanticError[] {
 
 function handlePydanticErrors(errors: PydanticError[]) {
   errors.map(({ type, loc, msg }) => {
-    const title = type
+    const message = typeof msg === "string" ? msg : Object.values(msg)[0];
+    if (!loc) {
+      toast.error(message);
+      return;
+    }
+    type = type
```

Comment:
not a huge fan of this, but conditionally approving for now. we need to improve how we handle errors later on, there are couple of edge cases missing here, and cases where that's not supposed to be handled here too.